polls/templates/polls/htmlform.html

```diff
@@ -9,24 +9,38 @@ <h3>Task</h3>
   <input type="hidden" value="{{ task_list }}" name="tasklist">
   <input type="hidden" value="{{ task_presentation }}" name="taskPresentation">
   <h4>ID: {{task.pk}}</h4>
-  <audio controls src="{{ reference_url }}" class="mt-3">
```

Comment:
At line 5 where the form tag is, if we're using htmx, the best way to declare the form is:

<form hx-post="{% url 'create_annotation' %}" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}' hx-boost="true">

Reply:
got it will change it